Python/import.c

```diff
@@ -4174,7 +4174,7 @@ _PyImport_FiniExternal(PyInterpreterState *interp)
 /******************/
 
 PyObject *
-_PyImport_GetModuleAttr(PyObject *modname, PyObject *attrname)
+PyImport_GetModuleAttr(PyObject *modname, PyObject *attrname)
```

Comment:
Hmm, PyObject_GetAttr can return NULL if the attrname is not a UnicodeObject, does not need to handle it?

Reply:
Ah well, it will return NULL in the end, so it will be fine :)

Reply:
Well spotted. I added tests on non-string module/attribute names. TypeError is raised in both cases.